### ISSUE TITLE
e_gauge and colors with e_add in case of timeline

### ISSUE THREAD

**Issue by rdatasculptor** [JohnCoene/echarts4r#352]
I know I can use `e_gauge_` if I need a gauge with a timeline. Now I am wondering if it is possible to give each group in the timeline its own gauge with its own color, using the parameter `axisLine`. 
I tried `e_add ` but the list structure is quite complex, and I am wondering if this is even possible. 

I know this not right, but maybe someone can me point in the right direction to deal with this:

```
e_add("axisLine",
             list(lineStyle = list(width=width,
                                              color = list(
                                                      list(0.25,  color1), 
                                                      list(0.5,  color2), 
                                                      list(0.75,  color3), 
                                                      list(1, color4))))
```

Many thanks in advance!
If needed I will make a better example.

**Comment by JohnCoene** [comment]
Yes indeed cannot be done currently I tried a few things and remembered I had to make a choice there on how these were processed and went with something that is not compatible with timeline.

**Comment by rdatasculptor** [comment]
Just a thought. Not sure if this is a good idea, but maybe something similar like you solved it in `e_timeline_serie()` when e.g. adding titles?
